### PR TITLE
Fix layout for iOS9,10

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
New call view layout had the top bar content too close to the top.

Also, I nudged the back button to where it normally resides.

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/CallViewController.swift [hunk 1]
```diff
@@ -503,11 +503,16 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
...
 
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .leading)
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+        leaveCallViewButton.autoPinEdge(toSuperviewEdge: .leading)
+
+        if #available(iOS 11, *) {
+            leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+            contactNameLabel.autoPinEdge(toSuperviewMargin: .top)
+        } else {
+            leaveCallViewButton.autoPin(toTopLayoutGuideOf: self, withInset: 0)
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/CallViewController.swift]: I actually don't know why I had to do this. =/

File: Signal/src/ViewControllers/CallViewController.swift [hunk 2]
```diff
@@ -503,11 +503,16 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
         // Dark blurred background.
         blurView.autoPinEdgesToSuperviewEdges()
 
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .leading)
-        leaveCallViewButton.autoPinEdge(toSuperviewMargin: .top)
+        leaveCallViewButton.autoPinEdge(toSuperviewEdge: .leading)
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/CallViewController.swift]: With the padding already in the image view, pinning to the margin moved the back button too far in.

File: Signal/src/ViewControllers/CallViewController.swift (hunk 1 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/CallViewController.swift]: Please maker sure to test this on iPhone X too if you haven't already.

**Comment by michaelkirk-signal** [comment]
In a881d57

**Pull request closed without merging**